nomination/templates/nomination/base.html

```diff
@@ -0,0 +1,34 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <title>{% block title %}{% endblock %}</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="shortcut icon" type="image/x-icon" href="{% static 'nomination/images/favicon.ico' %}" />
+        <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+        {% if project %}
+            <link rel="alternate" type="application/rss+xml" title="Newly Added URLs" href="{% url 'url_feed' project.project_slug %}" />
+            <link rel="alternate" type="application/rss+xml" title="Recent Nominations" href="{% url 'nomination_feed' project.project_slug %}" />
+        {% endif %}
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
```

Comment:
Update jquery link to grab newest version (1.12.4). Download minified file and place in scripts directory (https://code.jquery.com/jquery-1.12.4.min.js)

Add fallback code on line after CDN script load:
`<script>window.jQuery || document.write('<script src="{% static 'nomination/scripts/jquery-1.12.4.min.js' %}">\/script>')</script>`